applications/fishing-map/src/types/index.ts

```diff
@@ -40,3 +46,7 @@ export enum AsyncReducerStatus {
   Finished = 'finished',
   Error = 'error',
 }
+
+export type TimebarVisualisations = 'heatmap' | 'vessel'
+export type TimebarEvents = 'all' | 'fishing' | 'encounters' | 'loitering' | 'ports' | 'none'
+export type TimebarGraphs = 'speed' | 'depth' | 'none'
```

Comment:
(nitpick) I prefer enums than magic strings, but not important.
```suggestion
export enum TimebarGraphs {
  Speed = 'speed'
  Depth = 'depth,
  None = 'none',
}

```

(then use `TimebarGraphs.Speed` instead of 'speed', which will give you autocomplete)